src/engraving/rendering/score/tlayout.cpp

```diff
@@ -3762,12 +3765,62 @@ void TLayout::layoutLayoutBreak(const LayoutBreak* item, LayoutBreak::LayoutData
     LAYOUT_CALL_ITEM(item);
     LD_INDEPENDENT;
 
-    if (ldata->isValid()) {
+    if (ldata->isValid() || MScore::testMode) {
         return;
     }
 
-    double lw = item->lineWidth();
-    ldata->setBbox(item->iconBorderRect().adjusted(-lw, -lw, lw, lw));
+    FontMetrics metrics(item->font());
+    RectF bbox = metrics.boundingRect(item->iconCode());
+    ldata->setShape(Shape(bbox, item));
+
+    // Set z = 0 to ensure it stays behind notation
+    const_cast<LayoutBreak*>(item)->setZ(0); // TODO: z should be part of ldata
+}
+
+void TLayout::layoutSystemLockIndicator(const SystemLockIndicator* item, SystemLockIndicator::LayoutData* ldata)
+{
+    if (MScore::testMode) {
```

Comment:
Because this layout function needs to know about the Icon font, which is injected as part of the EngravingConfiguration module. But that module isn't loaded in tests, which means as soon as a test tries to layout one of these, it crashes (the same goes for LayoutBreaks which now use icons too). Or at least that's how I understand it. 

I did try to look into how to inject the module in the tests, but I couldn't understand it, so I gave up. And in general, given that these items are a bit hybrid (in that they are essentially UI elements drawn in the score) it probably doesn't make sense for that module to be loaded in the tests anyway, so I just avoided laying them out in test mode.

Reply:
That makes sense; perhaps add a brief comment to the code explaining that.